com.unity.uiwidgets/Runtime/engine/UIWidgetsPanel.cs

```diff
@@ -107,6 +107,7 @@ public class Configurations {
         }
     }
 
+    [ExecuteInEditMode]  
```

Comment:
what is the purpose to add this line?

Reply:
In Editor mode, coroutine is not excuted without it.

Reply:
ok